### PR TITLE
Added line and column numbers for error messages

### PR DESCRIPTION
**Pull request by hristoramirez**
Added error messages with line and column location 
pullrequest-line_column

### CODE REVIEW

File: webapp/src/errorList.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
             <div className="errorListInner" hidden={this.state.isCollapsed}>
                 {
                     (this.state.errors || []).map(e =>
-                        <div key={errorKey(e)}>{e.messageText}</div>)
+                        <div key={errorKey(e)}>{`${e.messageText} - [line ${e.line + 1}: col ${e.column}]`}</div>)
```
Comment (darzu) [inline webapp/src/errorList.tsx:47]: Looks great! Nice compact change

Comment (darzu) [reply webapp/src/errorList.tsx:47]: This is a good start and we can always tweak how things appear as we go

Comment (pelikhan) [reply webapp/src/errorList.tsx:47]: Unless we plan to translate line and col, you should remove them.

Comment (jwunderl) [reply webapp/src/errorList.tsx:47]: should this display `e.column + 1` like it does for for `e.line`? Just checked vs code and it gives both line and column indexed from 1 ~

**Comment by darzu** [comment]
> It would be good to format the errors in output.txt with the same format (future PR)

Good point, I'll file an item

**Pull request merged**